bootstrap/cmd/bootstrap/app/k8sAuth.go

```diff
@@ -41,6 +42,32 @@ func buildClusterConfig(ctx context.Context, token string, project string, zone
 	}, nil
 }
 
+// BuildClientCmdAPI takeks k8s config and access token, build and return clientcmdapi.Config entry
```

Comment:
kustomize uses kubernetes-v1.13.4, client-go-v10.0.0 - is there anything (other than ksonnet) that prevents bootstrap from using the the later APIs (I know that client-go requires some changes)?